### PR TITLE
Change list_display in DC Assset

### PR DESCRIPTION
**Pull request by szok**
- Change rack accessory list_display
- Change server room list_display
- Change rack list_display
- Change asset model list_display
- Change ip address list_display

Connected to #1759

### CODE REVIEW

File: src/ralph/data_center/admin.py [hunk 1]
```diff
@@ -189,13 +190,23 @@ class RackAccessoryInline(RalphTabularInline):
...
-    list_display = ['name', 'server_room']
+    list_display = ['name', 'server_room_name', 'data_center_name']
     list_filter = ['server_room__data_center']
     list_select_related = ['server_room', 'server_room__data_center']
     search_fields = ['name']
     inlines = [RackAccessoryInline]
     resource_class = resources.RackResource
 
+    def server_room_name(self, obj):
+        return obj.server_room.name
```
Comment (mkurek) [inline src/ralph/data_center/admin.py]: server room could be null

File: src/ralph/data_center/admin.py [hunk 2]
```diff
@@ -189,13 +190,23 @@ class RackAccessoryInline(RalphTabularInline):
...
+    server_room_name.short_description = _('Server room')
+    server_room_name.admin_order_field = 'server_room'
```
Comment (mkurek) [inline src/ralph/data_center/admin.py]: shouldn't it be `server_room__name` ?

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**